src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java

```diff
@@ -85,4 +86,34 @@ public void testUIElements() {
         Assert.assertTrue(uiElements.isOthersRadioButtonSelected(), "Others radio button was not selected!");
     }
 
+
+
+    @Test(description = "01")
+    @MethodOwner(owner = "ashchavinska")
+    public void verifyLoginPage(){
+        String username = "Rachel Green";
+        String password = "1234567890";
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertTrue(loginPage.isPageOpened(), "Login page isn't opened");
+        //verify fields are presented->they must be (name,password,male/female... already checked previously
```

Comment:
bad practice, isPageOpened must check only if page opened
if you need verify a lot of fields use one by one(one method for one field)
Example: isNameFieldPresented, ....

Reply:
done